contributor/zh/01-introduction-zh.asciidoc

```diff
@@ -6,13 +6,13 @@
 
 有了 InnerSource 项目，你将永远不必再为等待其他团队交付某些所需功能而困扰了。如果你没有从此项目中获取所需东西，则可以担任 InnerSource 的贡献者，直接在另一个团队的代码库中进行所需的更改。
 
-“贡献者”指为 InnerSource 社区项目代码库做贡献的人，这些人可以将自己视为社区的一部分。而对于大多数贡献者来说，都会经历这一系列过程：从了解社区，到使用社区的产品，再到与社区成员进行交互，最后开始做出贡献。最后，其中一些贡献者会成为 https://innersourcecommons.org/resources/learningpath/trusted-committer/zh/index[_Trusted Committer_]
+“贡献者”指为 InnerSource 社区项目代码库做贡献的人，这些人可以将自己视为社区的一部分。而对于大多数贡献者来说，都会经历这一系列过程：从了解社区，到使用社区的产品，再到与社区成员进行交互，最后开始做出贡献。最后，其中一些贡献者会成为 https://innersourcecommons.org/zh/learn/learning-path/trusted-committer/01[_Trusted Committer_]
```

Comment:
Do you think that we should just link to the landing page here so that they can see all of the content about Trusted Committers?

```suggestion
“贡献者”指为 InnerSource 社区项目代码库做贡献的人，这些人可以将自己视为社区的一部分。而对于大多数贡献者来说，都会经历这一系列过程：从了解社区，到使用社区的产品，再到与社区成员进行交互，最后开始做出贡献。最后，其中一些贡献者会成为 https://innersourcecommons.org/zh/learn/learning-path/trusted-committer/[_Trusted Committer_]
```

Reply:
Same for other `01` links.

Reply:
I agree with @rrrutledge . If we mention some specific roll of InnerSource, the landing page is better for the purpose.
There are many 01 links in articles, not only translations but also English articles.
Once a consensus made with others, I will make it in this PR.

Reply:
Others (@lenucksi / @tsadler1988 ?) chime in here, else we can discuss on the call.

Reply:
I agree with @ystk point. I'm in favor of changing those over.

Reply:
Thanks to join this discussion. I have a suggestion on this PR. 
If we are going to change 01 links for all articles which also includes English articles. I would like to suggest creating a new PR for that purpose.
I mean this PR is for fixing and other PR for a kind of policy changes.

Reply:
I merged a suggested change from @rrrutledge . The rest of the same links will be available in other PR.

Reply:
This is great, Yoshi.  We are very much looking forward to this.